config-store-api/pom.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<parent>
+		<artifactId>stargate</artifactId>
+		<groupId>io.stargate</groupId>
+		<version>0.0.18-SNAPSHOT</version>
+	</parent>
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>io.stargate.config-store</groupId>
+	<artifactId>config-store-api</artifactId>
+
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.apache.felix</groupId>
+				<artifactId>maven-bundle-plugin</artifactId>
+				<configuration>
+					<unpackBundle>true</unpackBundle>
+					<instructions>
+						<Bundle-Name>Config-Store-API</Bundle-Name>
+						<Bundle-Description>API for Config Store</Bundle-Description>
+						<Bundle-SymbolicName>io.stargate.config.store</Bundle-SymbolicName>
+						<Import-Package>org.osgi.framework</Import-Package>
+						<Export-Package><![CDATA[
+              io.stargate.config.store,
+            ]]></Export-Package>
```

Comment:
nit: odd formatting.... why not use zero indent for `io.stargate.config.store`?

Reply:
Alternatively, since we're not likely to export any other package here... do we really need `CDATA`?

Reply:
good catch, I will remove `CDATA`